woocommerce/woocommerce
# Shard e2e test execution on CI

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR tries to shard the execution of our e2e tests, executing them in 4 separate jobs so that there is less collision/flakiness.

Closes: #42833, #42713, #42003, #42739, #42982, #41817, #42723, #42832, #42744, #42966, #43278 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check that the CI runs/passes
2. Check that tests are merged at the end
3. Check that test results are still processed/posted



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
